frontend/src/Login/Signup.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+
+export default function Signup() {
```

Comment:
i see there's a decent amount of repetition between this component and the ``Login`` component (e.g. the welcome section, the username + password input section). can we try to think of some ways to combine these two components?

Reply:
Okay !